thundra/plugins/log/thundra_log_handler.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+logs = []
+
+
+class ThundraLogHandler(logging.Handler):
+    def __init__(self):
+        logging.Handler.__init__(self)
+        ThundraLogHandler.logs = []
+
+    def emit(self, record):
+        formatted_message = self.format(record)
+        log = {
+            'log': formatted_message,
+            'logMessage': record.msg,
+            'loggerName': record.name,
+            'logTimestamp': int(record.created * 1000),
+            'logLevel': record.levelname,
+            'logLevelId': record.levelno / 10
+        }
+        logs.append(log)
+
+
+logging.ThundraLogHandler = ThundraLogHandler
```

Comment:
Does this guarantee that `ThundraLogHandler` instance is singleton? If so do we need to define `logs` as global variable?